Makefile

```diff
@@ -2,8 +2,14 @@
 MKDOCS_MATERIAL_VERSION := 9.1.4
 PORT ?= 8000
 
+ifeq ($(shell command -v podman 2> /dev/null),)
```

Comment:
Why do you think podman should be the primary runtime?

Reply:
Because I have it :) (and I can test this option)
I have no strict opinion on if we should test for docker or podman. I'm happy to change it to docker, if you have a preference for that.

Reply:
Addressed in [320e60e](https://github.com/sdcio/docs/pull/67/commits/320e60ec4db851148011b0c762ae911fb3114cd2)